src/ontology/catalog-v001.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<catalog prefer="public" xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
+    <uri name="https:/www.inf.ufrgs.br/bdi/ontologies/releases/2024-04-05/imports/bfo2020_import.owl" uri="imports/bfo2020_import.owl"/>
```

Comment:
Here is probably where the problem with your import lies. For the catalog system to work correctly the following must all be the same:

1. The uri name in the catalog-v001.xml
2. the imports() statement in the geocore-edit.owl file
3. The ontology IRI in the `imports/bfo2020_import.owl`

Try to make them all the same in see what happens, hopefully your error goes away!

Reply:
I'm a bit confused here but I didn't really choose anything here. Maybe the origin of the problem is that at some time when opening Protégé it asked me to point to the import. Anyway, tried to do as you explained and got the error:
```
Run cd src/ontology && make ROBOT_ENV='ROBOT_JAVA_ARGS=-Xmx6G' test IMP=false PAT=false MIR=false
ODK Makefile v1.5
ODK Image v1.5
ROBOT version 1.9.5
Ammonite REPL & Script-Runner, 2.5.9
DOSDP-Tools v0.19.3
Konclude v0.[7](https://github.com/BDI-UFRGS/GeoCoreOntology/actions/runs/8581821886/job/23519241790#step:4:8).0-1138
Jena v4.9.0
Relation-Graph v2.3.2
WARNING: The directory '/github/home/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you should use sudo's -H flag.
Ontology Access Kit v0.5.25
robot --catalog catalog-v001.xml convert --input geocore-edit.owl --format ofn --output tmp/geocore-preprocess.owl
org.semanticweb.owlapi.model.UnloadableImportException: Could not load imported ontology: <https://www.inf.ufrgs.br/bdi/ontologies/geocore/imports/bfo2020_import.owl> Cause: /__w/GeoCoreOntology/GeoCoreOntology/src/ontology/imports/bfo2020_import.owl (No such file or directory)
Use the -vvv option to show the stack trace.
Use the --help option to see usage information.
make: *** [Makefile:32[9](https://github.com/BDI-UFRGS/GeoCoreOntology/actions/runs/8581821886/job/23519241790#step:4:10): tmp/geocore-preprocess.owl] Error 1
Error: Process completed with exit code 2.
```
The import in geocore-edit is:
```
Import(<https://www.inf.ufrgs.br/bdi/ontologies/geocore/imports/bfo2020_import.owl>)
```
Catalog is:
```
<uri name="https://www.inf.ufrgs.br/bdi/ontologies/geocore/imports/bfo2020_import.owl" uri="imports/bfo2020_import.owl"/>
```
bfo2020.owl is:
```
Ontology(<https://www.inf.ufrgs.br/bdi/ontologies/geocore/imports/bfo2020_import.owl>
<https://www.inf.ufrgs.br/bdi/ontologies/geocore/releases/2024-04-06/imports/bfo2020_import.owl>
```
So basically what changed is that I changed the imports to point to the ontology uri instead of the ontology version uri but still got the error. Is there anything that I'm overlooking here?

Reply:
Did you change the captialisation of the file name? bfo2020_import.owl instead of BFO2020_import.owl?

Reply:
Yes, I did but now that you mentioned I see that for some reason it isn't lower cased on github. I will try to remove and add the file again.

![image](https://github.com/BDI-UFRGS/GeoCoreOntology/assets/3798303/a43add2f-aeab-460e-8c94-7c7940b3de04)

Reply:
I cloned your repo and it works for me! And the action seems to pass at well! Good work!

Reply:
The problem was that I'm using Windows and git didn't notice the difference between upper/lower case! Thank you!